src/redux/books/books.js

```diff
@@ -0,0 +1,11 @@
+import Book from '../../components/book';
+import AddBooks from '../../components/addBooks';
+
+const Books = () => (
+  <div>
+    <Book />
+    <AddBooks />
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] Kindly extract this **Books** component into the components folder.

Reply:
![image](https://user-images.githubusercontent.com/50904584/145826509-c0bfad40-758a-4e49-a9dd-357934ceca99.png)

Print screen from the task description.
Could you please provide more info?